src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/IssueNumberRetriever.java

```diff
@@ -11,10 +11,14 @@
 @Service
 public class IssueNumberRetriever {
 
-    public static final String GET_NEXTVALUE_FROM_ISSUE_NUMBER_SEQUENCE = "select nextval('MAILING_ISSUE_NUMBER_SEQ') as issue";
+    private static final String GET_NEXTVALUE_FROM_ISSUE_NUMBER_SEQUENCE = "select nextval('MAILING_ISSUE_NUMBER_SEQ') as issue";
+    private static final String GET_CURRENT_FROM_ISSUE_NUMBER_SEQUENCE = "select last_value as issue from MAILING_ISSUE_NUMBER_SEQ";
 
-    private final DataSource dataSource;
+    private DataSource dataSource;
 
+    //Required byc Wicked when injected using @SpringBean
```

Comment:
WickeT :) // Not sure where is my previous comment about this. Code reviews in GH are far from being intuitive :)